crypto/keys/secp256r1/pubkey.go

```diff
@@ -0,0 +1,59 @@
+package secp256r1
+
+import (
+	tmcrypto "github.com/tendermint/tendermint/crypto"
+
+	ecdsa "github.com/cosmos/cosmos-sdk/crypto/keys/internal/ecdsa"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+)
+
+// String implements proto.Message interface.
+func (m *PubKey) String() string {
+	return m.Key.String(name)
+}
+
+// Bytes implements SDK PubKey interface.
+func (m *PubKey) Bytes() []byte {
+	return m.Key.Bytes()
+}
+
+// Equals implements SDK PubKey interface.
+func (m *PubKey) Equals(other cryptotypes.PubKey) bool {
+	pk2, ok := other.(*PubKey)
+	if !ok {
+		return false
+	}
+	return m.Key.Equal(&pk2.Key.PublicKey)
+}
+
+// Address implements SDK PubKey interface.
+func (m *PubKey) Address() tmcrypto.Address {
```

Comment:
Why not just sdk.Address?

Reply:
because there is no `sdk` package name. However I can change if there is a such desire.

Reply:
I have this desire, as I would like pubkeys to decouple from `tmcrypto` as much as possible

Reply:
tm would be `TM PubKey` interface ;)

I don't have a strong preference, so let's use `sdk.` :+1:

Reply:
miscommunication here, I thought that this is about  the comment not the `tmcrypto.Address` type. 
The reason I used `tmcrypto.Address` here is because we used it in `secp256k1`. I will update the type here and the secp256k1 in other PR.

Reply:
I have more thoughts about it. The `tmcrypto.Address` is used in all other keys. So I prefer to be consistent here and solve that issue here https://github.com/cosmos/cosmos-sdk/issues/8775  (I'm more in favor of removing the SDK Address type).